packages/lesswrong/lib/collections/revisions/collection.ts

```diff
@@ -15,9 +15,10 @@ export const Revisions: RevisionsCollection = createCollection({
   typeName: 'Revision',
   schema,
   resolvers: getDefaultResolvers('Revisions'),
-  // No mutations (revisions are insert-only immutable, and are created as a
-  // byproduct of creating/editing documents in other collections).
-  // mutations: getDefaultMutations('Revisions'),
+  // This has mutators because of a few mutable metadata fields (eg
+  // skipAttributions), but most parts of revisions are create-only immutable.
+  mutations: getDefaultMutations('Revisions'),
```

Comment:
Hm, maybe we only want to allow users to do updates and not creates?  (Also we probably want to forbid deletes entirely.)